src/p_mobj.c

```diff
@@ -1302,24 +1349,28 @@ void P_SpawnMapThing (mapthing_t* mthing)
 
 extern fixed_t attackrange;
 
-void P_SpawnPuff(fixed_t x,fixed_t y,fixed_t z)
+void P_SpawnPuff(fixed_t x, fixed_t y, fixed_t z)
+{
+    return P_SpawnPuffSafe(x, y, z, false);
```

Comment:
The `return` will have to get removed. MSVC does not allow for `void` functions to return `void` functions.